Cargo.toml

```diff
@@ -12,3 +12,4 @@ lto = true
 
 [patch.crates-io]
 kvm-bindings = { git = "https://github.com/firecracker-microvm/kvm-bindings", tag = "v0.6.0-2", features = ["fam-wrappers"] }
+userfaultfd = { git = "https://github.com/bchalios/userfaultfd-rs", branch = "kernel-6.1" }
```

Comment:
I assume you will open a PR to upstream `userfaultfd-rs`. Otherwise it is OK to rely on the for for now, but then it might be worth adding `rev="..."` to fix the commit.

Reply:
Yes, once we have the code upstream, I will drop this. That's why the PR is not Draft

Reply:
updated to the new version of userfaultfd-rs